pkg/branding/brandedtext_test.go

```diff
@@ -5,16 +5,32 @@ import (
 
 	"github.com/stackrox/rox/pkg/testutils/envisolator"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
 )
 
 const (
 	brandedProductNameRHACS    = "Red Hat Advanced Cluster Security for Kubernetes"
 	brandedProductNameStackrox = "StackRox"
 )
 
-func TestGetBrandedProductName(t *testing.T) {
-	envIsolator := envisolator.NewEnvIsolator(t)
+func TestBrandedText(t *testing.T) {
+	suite.Run(t, new(BrandedTextTestSuite))
+}
+
+type BrandedTextTestSuite struct {
+	suite.Suite
+	envIsolator *envisolator.EnvIsolator
+}
+
+func (s *BrandedTextTestSuite) SetupSuite() {
+	s.envIsolator = envisolator.NewEnvIsolator(s.T())
+}
+
+func (s *BrandedTextTestSuite) TeardownTest() {
```

Comment:
I think this isn't being executed. It's supposed to be `TearDownTest`. Have a look at the [docs](https://pkg.go.dev/github.com/stretchr/testify/suite#TearDownTestSuite). 

Go interfaces aren't enforced -- so this is just a method definition that isn't called by testify. 

@msugakov and I had a vendetta against all these occurrences. One way to do type assertions on this is to cast your struct to each interface your're implementing. Similar to what we're doing [here](https://github.com/stackrox/stackrox/blob/master/sensor/common/detector/networkpolicy/networkpolicy_test.go#L28-L29). If you do this, the compiler will throw an error if the interface is incorrectly implemented.